Text-Tools.md

```diff
@@ -113,30 +113,12 @@
 ## ▷ Grammar Check
 
 * ⭐ **[Scribens](https://www.scribens.com/)**
-* ⭐ **[Jenni](https://jenni.ai/)**
-* [Wordtune](https://www.wordtune.com/)
-* [HemingwayApp](https://hemingwayapp.com/)
-* [Grammark](https://grammark.org/)
-* [SlickWrite](https://www.slickwrite.com/)
-* [Ginger Software](https://www.gingersoftware.com/grammarcheck)
 * [LanguageTool](https://languagetool.org/)

```

Comment:
Max 9997 chars

Reply:
yeah but otherwise free and unlimited, you can paste text in parts